packages/core/src/html/includePanelProcessor.ts

```diff
@@ -230,6 +230,24 @@ export function processInclude(node: MbNode, context: Context, pageSources: Page
     const $ = cheerio.load(actualContent);
     const actualContentOrNull = $(hash).html();
     actualContent = actualContentOrNull || '';
+    if (actualContent !== '') {
+      const footnotePattern = /<a aria-describedby="footnote-label" href="#fn-\d+-\d+">\[\d+\]/g;
```

Comment:
oh also small thing but i was wondering if using cheerio here makes more sense?

Reply:
Emm, but is not to use cheerio, we need the exact id (eg: `#${href}.footnote-item`)? So we still need to pattern matching (and also to replace the index in the original content).

Reply:
Yeah I get that cheerio will do the same thing but I think its cleaner to use cheerio to select out and append. In that case, we can use cheerio node to do the modifications below as well. What do you think?

Reply:
@yucheng11122017 
Thank you for the clarification; I have explored cheerio apis and made the requested changes. Following is a quick snapshot:
<img width="699" alt="Screenshot 2024-03-14 at 23 47 04" src="https://github.com/MarkBind/markbind/assets/121547057/14e5e45a-0681-4ff4-bbe8-bd2f372baeec">

Reply:
(console.log on last line was removed in later commits)